docs/recipes/precompiling-with-webpack.md

```diff
@@ -0,0 +1,29 @@
+## Precompiling source files with webpack
+
+The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) mentions precompiling your imported modules as an alternative to runtime compilation, but it doesn't explain how. This recipe shows how to do this using webpack.
+
+###### webpack.config.js
+
+```js
+const nodeExternals = require('webpack-node-externals');
+
+module.exports = {
+	entry: ['src/tests.js'],
+	target: 'node',
+	output: {
+		path: '_build',
+		filename: 'tests.js'
```

Comment:
As someone who didn't know about this feature, I'm asking questions from a first time user standpoint that I think are important to address: 

*  I would love to see an example of what this file looks like. Could a code block be added above to show this? Is it just a list of `t.test`? Or better yet is there a repo for this?